Update rdm chapter (#4556)
the-turing-way/the-turing-way: book/website/reproducible-research/rdm/rdm-visualisation.md

- [Tips to improve interpretability and accessibility](https://www.youtube.com/watch?v=RzT95DVUMnw) by Dr Tracey Weissgerber (video)
- [Writing Meaningful Alt-Texts for Data Visualisations in R](https://www.youtube.com/watch?v=dXV5bx1WQTM)
- [Writing Alt Text to communicate the meaning in data visualizations] by @Hare2022dataviz
- [Alt-texts](https://axesslab.com/alt-texts/)
- [Writing Alt Text](https://dataabinitio.com/?p=1161) for a Scientific Figure by Kristin Briney

> - [Writing Alt Text for Data Visualization](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81) (To get started, use Amy Cesal's quick formula: [alt text = Chart type of type of data where reason for including chart. Link to data.])

Reviewer: ```suggestion
- [Writing Alt Text for Data Visualization](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81)
  To get started, use Amy Cesal's quick formula: [alt text = `chart type` of `type of data` where `reason for including chart`. `Link to data`.]
```

Seeing if I can get this sentence to look a little clearer.... please feel free to disagree and/or change!

- [Writing Meaningful Alt-Texts for Data Visualisations in R](https://www.youtube.com/watch?v=dXV5bx1WQTM)

> - [Writing Alt Text to communicate the meaning in data visualizations] by {cite:ps}`Hare2022dataviz`

Author: ```suggestion
- [Writing Alt Text to communicate the meaning in data visualizations] by @Hare2022dataviz
```

- [Data Visualisation, A practical introduction](https://socviz.co/) by {cite:ps}`Healy2018dataviz`
- [Creating clear and informative image-based figures for scientific publications](https://doi.org/10.1371/journal.pbio.3001161) by {cite:ps}`Jambor2021dataviz`

> - [A layered grammar of graphics](http://dx.doi.org/10.1198/jcgs.2009.07098) by {cite:ps}`Wickham2010dataviz`

Reviewer: These are old referencing styles. Could be updated if you feel like it!

I'm not 100% sure these are the right formats - please check!

```suggestion
- Data Visualisation, A practical introduction by @Healy2018dataviz
- Creating clear and informative image-based figures for scientific publications by @Jambor2021dataviz
- A layered grammar of graphics by @Wickham2010dataviz
```

- A Field Guide to Digital Color by {cite:ps}`Stone2003dataviz`
- The Grammar of Graphics by {cite:ps}`Wilkinson1999dataviz`

> - The Visual Display of Quantitative Information by {cite:ps}`Tufte2001dataviz`

Author: ```suggestion
- The Visual Display of Quantitative Information by @Tufte2001dataviz
```
